packages/lib/types/document-with-recipient.ts

```diff
@@ -0,0 +1,5 @@
+import { Document, Recipient } from '@documenso/prisma/client';
```

Comment:
This could probably sit somewhere in the prisma package since it relates directly to database types